packages/alfa-rules/src/sia-r81/rule.ts

```diff
@@ -143,10 +143,9 @@ function linkContext(element: Element, device: Device): Set<dom.Node> {
   }
 
   for (const describedby of element.attribute("aria-describedby")) {
-    for (const reference of element
-      .root()
-      .elementDescendants()
-      .filter(hasId(equals(...describedby.tokens())))) {
+    for (const reference of getElementDescendants(element.root()).filter(
+      hasId(equals(...describedby.tokens()))
+    )) {
```

Comment:
**Suggestion:** This actually looks like it could use `getIdMap` 🤔 
It seems to be fetching all elemenjt whose `id` is one of the `describedby.tokens`.

Reply:
Cool, good catch. I'll look into it 😃